src/bm_sim/options_parse.cpp

```diff
@@ -18,22 +18,22 @@
  *
  */
 
-#include <bm/config.h>
 #include <bm/bm_sim/options_parse.h>
+
```

Comment:
I think this is another instance where includes should be re-ordered instead of adding an empty line?